Dockerfile

```diff
@@ -28,7 +29,7 @@ COPY boot /sbin/
 COPY fake-pups /pups/bin/pups
 
 ADD https://github.com/discourse/socketee/releases/download/v0.0.2/socketee /usr/local/bin/
-RUN echo '7cd6df7aeeac0cce35c84e842b3cda5a4c36a301  /usr/local/bin/socketee' | sha1sum -c - \
-	&& chmod 0755 /usr/local/bin/socketee
+RUN echo '7cd6df7aeeac0cce35c84e842b3cda5a4c36a301  /usr/local/bin/socketee' | sha1sum -c -
+RUN chmod 0755 /usr/local/bin/socketee
```

Comment:
Was there a reason to remove the `&&` and replace it with a new `RUN`?

This change will ask Docker to create, and cache, a separate layer.  The new cached layer will not provide any performance improvement because it will always be invalidated if there is any change to the preceding lines.  This is why you often see long `&&` chains in `RUN`.  The objective is to produce a single layer that will sometimes prove useful in cache.  (Were I writing this from scratch, I'd probably go so far as to replace the `ADD` with `RUN curl ...` just to remove that redundant layer, too.)

Refer to our internal haproxy `Dockerfile` for an example.  There, you will see separate `RUN` instructions that are designed to maximise use of the layer cache without adding redundant layers that would always be invalidated.

References:
- https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
- https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache

> In older versions of Docker, it was important that you minimized the number of layers in your images to ensure they were performant.

Perhaps this is less of a problem now, but it would still seem sensible to avoid writing unnecessary layers when they serve no benefit to the cache.